test/benchmark/lmbench/tcp_loopback_lat/run.sh

```diff
@@ -6,6 +6,6 @@ set -e
 
 echo "*** Running lmbench TCP latency test ***"
 
-/benchmark/bin/lmbench/lat_tcp -s
+/benchmark/bin/lmbench/lat_tcp -s 127.0.0.1
```

Comment:
What's the difference of -s and -S? Seems to be redundant.

Reply:
`-s` means starting a server on the given address, while `-S` means shutting down the server on the given address. 

For the newly added test, since the server is in the VM, and we call `sleep 200` after starting the server, so there's no need to shut down the server in the VM.